xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -2862,6 +2862,10 @@ bool CLinuxRendererGL::Supports(ESCALINGMETHOD method)
     if (scaleX < minScale && scaleY < minScale)
       return false;
 
+    // avoid using hq scalers when downscaling
```

Comment:
I don't think that this is wanted for all cases. Assume I run Kodi in windowed mode on my laptop. Why do you want me to have lower quality in this case?